charts/scalardl-auditor-monitoring/values.yaml

```diff
@@ -0,0 +1,693 @@
+    auth_enabled: false
+    extraMemberlistConfig:
+      bind_addr:
+        - ${MY_POD_IP}
+    commonConfig:
+      replication_factor: 1
+    schemaConfig:
+      configs:
+        - from: "2024-04-01"
+          store: tsdb
+          object_store: s3
+          schema: v13
+          index:
+            prefix: loki_index_
+            period: 24h
+    pattern_ingester:
+      enabled: true
+    limits_config:
+      retention_period: 720h
+      discover_service_name: []
+    compactor:
+      retention_enabled: true
+      delete_request_store: s3
+
+  deploymentMode: SingleBinary
+
+  singleBinary:
+    replicas: 1
+    extraArgs:
+      - -config.expand-env=true
+    extraEnv:
+      - name: MY_POD_IP
+        valueFrom:
+          fieldRef:
+            fieldPath: status.podIP
+
+  minio:
+    enabled: true
+    additionalLabels:
+      app.kubernetes.io/app: "scalardl-auditor-monitoring"
+    podLabels:
+      app.kubernetes.io/app: "scalardl-auditor-monitoring"
+    containerSecurityContext:
+      readOnlyRootFilesystem: false
+      capabilities:
+        drop:
+          - ALL
+      allowPrivilegeEscalation: false
+    postJob:
+      securityContext:
+        enabled: true
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+    makeBucketJob:
+      securityContext:
+        enabled: true
+      containerSecurityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        capabilities:
+          drop:
+            - ALL
+        allowPrivilegeEscalation: false
+    makeUserJob:
+      securityContext:
+        enabled: true
+      containerSecurityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        capabilities:
+          drop:
+            - ALL
+        allowPrivilegeEscalation: false
+
+  test:
+    enabled: false
+  lokiCanary:
+    enabled: false
+  gateway:
+    enabled: false
+  resultsCache:
+    enabled: false
+  chunksCache:
+    enabled: false
+
+  backend:
+    replicas: 0
+  read:
+    replicas: 0
+  write:
+    replicas: 0
+  ingester:
+    replicas: 0
+  querier:
+    replicas: 0
+  queryFrontend:
+    replicas: 0
+  queryScheduler:
+    replicas: 0
+  distributor:
+    replicas: 0
+  compactor:
+    replicas: 0
+  indexGateway:
+    replicas: 0
+  bloomCompactor:
+    replicas: 0
+  bloomGateway:
+    replicas: 0
+
+alloy:
+  enabled: true
+
+  global:
+    podSecurityContext:
+      seccompProfile:
+        type: RuntimeDefault
+
+  controller:
+    podLabels:
+      app.kubernetes.io/app: "scalardl-auditor-monitoring"
+      container.kubeaudit.io/alloy.allow-run-as-root: ""
+    type: "deployment"
+    replicas: 3
+
+  configReloader:
+    securityContext:
+      capabilities:
+        drop:
+          - ALL
+      allowPrivilegeEscalation: false
+      runAsNonRoot: true
+
+  alloy:
+    securityContext:
+      capabilities:
+        drop:
+          - ALL
+      allowPrivilegeEscalation: false
+    configMap:
+      content: |
```

Comment:
This config includes the following settings by default:

- Collect logs from ScalarDL Auditor related pods
  - ScalarDL Auditor
  - Envoy
  - Scalar Admin for Kubernetes
- Send logs to Loki

Also, it collects logs only in the same namespace that Alloy is deployed.